Lesson 3/cart.js

```diff
@@ -0,0 +1,68 @@
+function sendCartRequest(url) {
+    return new Promise((resolve, reject) => {
+        const xhr = new XMLHttpRequest();
+        xhr.open('GET', url);
+
+        xhr.onreadystatechange = function () {
+            if (xhr.readyState === XMLHttpRequest.DONE) {
+                if (xhr.status !== 200) {
+                    reject();
+                }
+                const users = JSON.parse(xhr.responseText);
+
+                resolve(users);
+            }
+        };
+        xhr.send();
+    });
+}
+
+class cartItemsList {
```

Comment:
Названия классов должны быть с заглавной буквы
```suggestion
class CartItemsList {
```